src/devtools/client/debugger/src/utils/editor/line-events.ts

```diff
@@ -64,7 +63,11 @@ function emitLineMouseEnter(codeMirror: $FixTypeLater, target: HTMLElement) {
     }
   );
 
-  dispatch(codeMirror, "lineMouseEnter", { lineNumber, lineNode, lineNumberNode });
+  dispatch(codeMirror, "lineMouseEnter", {
+    columnIndex,
+    lineNumber: lineIndex + 1,
+    lineNumberNode,
+  });
```

Comment:
There are only two places in the code base listening for this event, and neither was using the `lineNode` param.

https://github.com/replayio/devtools/blob/0e631c29360ef690def2654f193daf72c1839f87/src/devtools/client/debugger/src/components/Editor/ToggleWidgetButton.tsx#L199-L215

https://github.com/replayio/devtools/blob/main/src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx#L123-L142